nvcc-10/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM nvidia/cuda:10.1-devel-ubuntu16.04
+
+LABEL maintainer="David Beckingsale <david@llnl.gov>"
+
+ENV CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda-9.1
```

Comment:
Is this 9 or 10? or does nvcc not match the cuda version number?

Reply:
They should match. Also, note that cuda 10 is required now with the version of Google Test that comes with BLT.